src/dotnet-interactive/CommandLine/BrowserFrontendEnvironment.cs

```diff
@@ -8,5 +8,6 @@ namespace Microsoft.DotNet.Interactive.App.CommandLine
     public class BrowserFrontendEnvironment : FrontendEnvironment
     {
         public Uri ApiUri { get; set; }
+        public string Flavor { get; set; }
```

Comment:
This is pretty unclear in terms of how it should be understood or used.

Reply:
agreed